tests/integrations/tso/consistency_test.go

```diff
@@ -27,7 +27,7 @@ import (
 	tso "github.com/tikv/pd/pkg/mcs/tso/server"
 	tsopkg "github.com/tikv/pd/pkg/tso"
 	"github.com/tikv/pd/pkg/utils/tempurl"
-	pd "github.com/tikv/pd/pkg/utils/testutil"
+	tu "github.com/tikv/pd/pkg/utils/testutil"
```

Comment:
Why not just use the original package name?

Reply:
In most places, it is called `tu`